fe/fe-core/src/main/java/com/starrocks/metric/MetricRepo.java

```diff
@@ -182,15 +182,21 @@ public Integer getValue() {
                 "max_journal_id", MetricUnit.NOUNIT, "max journal id of this frontends") {
             @Override
             public Long getValue() {
-                EditLog editLog = Catalog.getCurrentCatalog().getEditLog();
-                if (editLog == null) {
-                    return -1L;
-                }
-                return editLog.getMaxJournalId();
+                return Catalog.getCurrentCatalog().getMaxJournalId();
             }
         };
         STARROCKS_METRIC_REGISTER.addMetric(maxJournalId);
 
+        // meta log total count
+        GaugeMetric<Long> metaLogCount = new GaugeMetric<Long>(
+                "meta_log_count", MetricUnit.NOUNIT, "meta log total count") {
+            @Override
+            public Long getValue() {
+                return Catalog.getCurrentCatalog().getMaxJournalId() - Catalog.getCurrentCatalog().getImageJournalId();
```

Comment:
why not use edit.getMaxJournalId() - edit.getMinJournalId()

Reply:
editLog.getMinJournalId() is not same to imageJournalId in some situation. Bdb log will be deleted only if all fes get the new image file. So when there are crashed fe, the editLog.getMinJournalId() will be bigger than imageJournalId.

Reply:
the editLog.getMinJournalId() will be **smaller** than imageJournalId.